src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
```

Comment:
Yes, you could get away without this.  You could, for example, get the number of psf components from each patch in ```fit_object_psfs!```, get it from the dimension of ```star_mcs``` and ```gal_mcs``` in ```populate_fsm_vecs```, pass it into ```fit_object_psfs!``` as an argument, &c.  But then, the same could be said of ```ElboArgs.S``` and ```ElboArgs.N```.  This is readable and consistent with the paradigm that model parameters go in ```ElboArgs```.